week3/ex4_permutations_final.py

```diff
@@ -0,0 +1,20 @@
+def perm(lst):
```

Comment:
Why you have two identical files? Please remove one.

Reply:
done